tesseract-server/Cargo.toml

```diff
@@ -17,6 +17,7 @@ pretty_env_logger = "0.2.4"
 serde = "1.0.70"
 serde_derive = "1.0.70"
 serde_qs = "0.4.1"
+serde_urlencoded = "0.5.1"
```

Comment:
I thought serde_qs will also do the deserialization of query strings? (which is what I think you're using this for)